File: src/Coor.js
class Coor {
  constructor(X, Y) {
    this.X = X;
    this.Y = Y;
  }

  copy() {
    return new Coor(X, Y);
  }

  equal(c) {
    return this.X == c.X && this.Y == c.Y;
  }

  debugCoor() {
    System.out.println("Coord: (" + X + ", " + Y + ")");
  }
}

File: src/backend/src/main/java/com/tucil3/backend/lib/Coor.java
package com.tucil3.backend.lib;
public class Coor {
    public int X;
    public int Y;

    public Coor(int Y, int X) {
        this.X = X;
        this.Y = Y;
    }

    public Coor copy() {
        return new Coor(Y, X);
    }
    
    public boolean equal(Coor c) {
        return this.X == c.X && this.Y == c.Y;
    }

    public void debugCoor() {
        System.out.println("Coord: (" + X + ", " + Y + ")");
    }
}
